src/sql/parts/accountManagement/firewallRuleDialog/firewallRuleDialog.ts

```diff
@@ -93,17 +93,19 @@ export class FirewallRuleDialog extends Modal {
 
 	protected renderBody(container: HTMLElement) {
 		let descriptionSection;
-		$().div({ class: 'firewall-rule-description-section new-section' }, (descriptionContainer) => {
+		$().div({ 'class': 'firewall-rule-description-section new-section' }, (descriptionContainer) => {
 			descriptionSection = descriptionContainer.getHTMLElement();
 			DOM.append(descriptionContainer.getHTMLElement(), DOM.$('div.firewall-rule-icon'));
 
 			const textDescriptionContainer = DOM.append(descriptionContainer.getHTMLElement(), DOM.$('div.firewall-rule-description'));
 			let dialogDescription = localize('firewallRuleDialogDescription',
 				'Your client IP address does not have access to the server. Sign in to an Azure account and create a new firewall rule to enable access.');
 			this.createLabelElement(new Builder(textDescriptionContainer), dialogDescription, false);
-			this._helpLink = DOM.append(textDescriptionContainer, DOM.$('a.help-link'));
-			this._helpLink.setAttribute('href', 'https://docs.microsoft.com/en-us/azure/sql-database/sql-database-firewall-configure');
-			this._helpLink.innerHTML += localize('firewallRuleHelpDescription', 'Learn more about firewall settings');
+
+			// TODO: Make this 1) extensible and 2) open the info via an action
```

Comment:
could you please add a date to this comment?  I find this useful when I see "dead code" so a couple years later I know it's safe to delete 😄

Reply:
Sure thing! I normally link to the associated issue, too, but I forgot to do it this time.